cmd/token/create.go

```diff
@@ -82,3 +83,11 @@ k0s token create --role worker --expiry 10m  //sets expiration time to 10 minute
 
 	return cmd
 }
+
+func checkCreateTokenRole(cmd *cobra.Command, args []string) error {
+	if !(createTokenRole == controllerRole || createTokenRole == workerRole) {
```

Comment:
IMO the negated or'ed conditions is harder to fit into ones head, at least into mine, than having plain:
```
if createTokenRole != controllerRole && createTokenRole != workerRole) {
```

Reply:
Linter doesn't like it:
```
suspect or: listTokenRole != controllerRole || listTokenRole != workerRole
```
but you are right i will change the "if"